packages/cart/src/components/Checkbox/styles.ts

```diff
@@ -0,0 +1,40 @@
+import styled from 'styled-components';
+
+const Input = styled.input`
+  border: 0;
+  clip: rect(0px, 0px, 0px, 0px);
+  height: 1px;
+  width: 1px;
+  padding: 0;
+  margin: 0;
+  overflow: hidden;
+  white-space: nowrap;
+  position: absolute;
+`;
```

Comment:
```suggestion
const Input = styled.input`
  display: none;
`;
```

`display: none`으로 해도 체크박스는 일 잘합니더!

Reply:
반영 완료! e866ada